ui/components/app/asset-list-item/asset-list-item.js

```diff
@@ -92,7 +92,7 @@ const AssetListItem = ({
 
   return (
     <ListItem
-      className={classnames('asset-list-item', className)}
+      className={classnames('asset-list-item', className, !secondary ? 'no-subtitle' : '')}
```

Comment:
could we follow BEM syntax here so asset-list-item--no-subtitle (as a BEM modifier)